src/modules/dashboard/components/HookedColonyAvatar/HookedColonyAvatar.ts

```diff
@@ -1,10 +1,9 @@
 import ColonyAvatar, { Props as ColonyAvatarProps } from '~core/ColonyAvatar';
-import { useDataFetcher, useDataSubscriber } from '~utils/hooks';
+import { useDataFetcher } from '~utils/hooks';
 import { withHooks } from '~utils/hoc';
-import { AnyColony } from '~data/index';
+import { AnyColony, useColonyQuery } from '~data/index';
```

Comment:
Shoul `AnyColony` here be replaced back with `ColonyType` ?

Reply:
No, in this case it's fine, we just need the colony from the DB